src/js-marketing-notification.ts

```diff
@@ -89,11 +89,8 @@ export class JsMarketingNotification {
     const expansionClass = 'marketing-notification--expanded';
     node.classList.toggle(expansionClass);
 
-    if (node.classList.contains(expansionClass)) {
-      if (this.openCallback) this.openCallback(node);
-    }
-    else {
-      if (this.closeCallback) this.closeCallback(node);
-    }
+    if (node.classList.contains(expansionClass) && this.openCallback) return this.openCallback(node);
```

Comment:
Shouldn't need to check the validity of `this.openCallback` as your setter above should handle it. This should be call-safe.

We also don't need to return anything, as callbacks should just be `void`. So, doing `if (node.classList.contains(expansionClass) this.openCallback(node);` should suffice.